packages/components/src/presentation-components/properties/inputs/AsyncSelect.tsx

```diff
@@ -77,10 +77,16 @@ function NoOptionsMessage<TOption, IsMulti extends boolean = boolean>({ children
 }
 
 function ValueContainer<TOption, IsMulti extends boolean = boolean>({ children, ...props }: ValueContainerProps<TOption, IsMulti>) {
+  const childrenArray = Children.toArray(children);
+  const values = childrenArray.slice(0, -1);
```

Comment:
I think we should check how many values are selected.

Reply:
Sliced the children based on selected option count.